apps/system/i2c/Makefile

```diff
@@ -125,8 +127,17 @@ install:
 
 endif
 
+ifeq ($(CONFIG_SYSTEM_I2CTOOL),y)
```

Comment:
This is only when CONFIG_BUILTIN_APPS is enabled.
let's change to ifeq ($(CONFIG_BUILTIN_APPS),$(CONFIG_SYSTEM_I2CTOOL),yy)